content/frameworks/react/getting-started/index.md

```diff
@@ -0,0 +1,97 @@
+
+```bash
+npx create-react-app new-app
+```
+
+<div class="aside note">
+<p><code>npx</code> is a package runner that is installed automatically with <code>npm</code> 5.2.0 or later. It simplifies quite a few processes involved with managing packages including running CLI commands (like <code>create-react-app</code> without having to install it globally on your machine.</p>
+
+<p>Take a look at <a href="https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b">Introducing npx: an npm package runner</a> if you would like to learn more.</p>
+</div>
+
+Once the command finishes executing, you can navigate to and run the application with the following commands:
+
+```bash
+cd new-app
+npm start
+```
+
+The following embed shows the directory structure and actual web page of a newly bootstrapped CRA application.
+
+<div class="glitch-embed-wrap" style="height: 346px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/new-create-react-app?path=src/App.js&attributionHidden=true"
+    alt="new-create-react-app on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+<div class="aside note">
+  Although CRA makes it simple to get started building a React application, there are many different ways to begin building sites with React. The <a href="https://github.com/facebook/create-react-app">"Popular Alternatives"</a> section within the README covers a few different cases where you might want to go with a different option.
+</div>
+
+There are multiple configuration files and build scripts that CRA uses to set up a webpack and Babel build process that includes a base [Jest](https://jestjs.io/) setup for testing. To make things simpler for the user, these files are hidden and cannot be accessed until you eject from CRA with `npm run eject`. This lets you take control of these configuration files yourself but once you eject, you are on your own.
+
+To give you a better idea of the many configuration files, here is the directory structure of an ejected CRA application:
+
+<div class="glitch-embed-wrap" style="height: 346px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/ejected-create-react-app?path=package.json&previewSize=0&attributionHidden=true"
+    alt="ejected-create-react-app on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+<div class="aside note">
+  Although it can be useful to know how all the different configuration files underneath the Create React App hood works, we will not need to eject or dive into any of them for the guides and codelabs in this learning path. 
+</div>
+
+## What's next?
+
+Now that you have a decent understanding of how every React application in this learning path is bootstrapped, each separate guide that follows this will cover a different topic that explains how to improve the performance of a specific part of your React site. This includes:
```

Comment:
Removing some words: "Now that you know how to get started building a Create React App, learn how improve your app's performance, including:"

Reply:
Updated